compiler/luci-interpreter/src/kernels/FullyConnected.test.cpp

```diff
@@ -128,6 +128,45 @@ void Check<uint8_t>(
               FloatArrayNear(output_data, quantized_tolerance));
 }
 
+void CheckHybridS4(std::initializer_list<int32_t> input_shape,
+                   std::initializer_list<int32_t> weights_shape,
+                   std::initializer_list<int32_t> bias_shape,
+                   std::initializer_list<int32_t> output_shape,
+                   std::initializer_list<float> input_data,
+                   std::initializer_list<int8_t> weights_initializer,
+                   std::initializer_list<float> bias_data, std::initializer_list<float> output_data)
+{
+  std::unique_ptr<IMemoryManager> memory_manager = std::make_unique<TestMemoryManager>();
+
+  Tensor input_tensor =
+    makeInputTensor<DataType::FLOAT32>(input_shape, input_data, memory_manager.get());
+  std::vector<int8_t> quantized_data(weights_initializer);
+  Tensor weights_tensor(DataType::S4, weights_shape, {{0.5}, {0}}, "");
+  memory_manager->allocate_memory(weights_tensor);
+  weights_tensor.writeData(quantized_data.data(), quantized_data.size() * sizeof(int8_t));
+  Tensor weights_scratch(DataType::FLOAT32, weights_shape, {}, "");
+  memory_manager->allocate_memory(weights_scratch);
+
+  Tensor bias_tensor =
+    makeInputTensor<DataType::FLOAT32>(bias_shape, bias_data, memory_manager.get());
+  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+
+  const float quantized_tolerance = getTolerance(-127, 128, 255);
```

Comment:
Can you add some comments why these values are like this?

Reply:
scale = 0.5
zerop = 0

quantize_steps = 15
min = -3.75
max = 3.75

Reply:
Ah.. please remove https://github.com/Samsung/ONE/pull/12801#discussion_r1538493500. `quantized_tolerance` is for output. scale 0.5 is for weights.